web/pgadmin/static/js/helpers/Notifier.jsx

```diff
@@ -119,34 +120,35 @@ AlertContent.propTypes = {
 
 
 let Notifier = {
-  success(msg, autoHideDuration = AUTO_HIDE_DURATION) {
-    this._callNotify(msg, MESSAGE_TYPE.SUCCESS, autoHideDuration);
+  success(msg, autoHideDuration = AUTO_HIDE_DURATION, persist = PERSIST_SNACK_BAR) {
```

Comment:
We don't need these changes. Just pass the autoHideDuration as null. 
https://github.com/iamhosseindhv/notistack/blob/v1.0.10/src/index.d.ts#L125